index.html

```diff
@@ -310,18 +317,21 @@ <h2>Skills</h2>
           </ul>
         </div>
       </div>
-      <p class="footer-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in
-        touch.</p>
     </section>
     <section class="contact-form">
-      <img class="contact-bg" src="./images/contact-bg.svg" alt="contact-form-background">
       <div class="container">
-        <form class="flex" action="https://formspree.io/f/meqvpklr" method="post">
-          <input type="text" name="full-name" placeholder="Full name" maxlength="30" required>
-          <input type="email" name="email" placeholder="Email address" required>
-          <textarea name="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
-          <input type="submit" value="Get in touch">
-        </form>
+        <div class="two-pane">
+          <p class="footer-text">I'm always interested in hearing about new projects, so if you'd like to chat please
+            get in
+            touch.</p>
+          <form class="flex" action="https://formspree.io/f/meqvpklr" method="post">
+            <input type="text" name="full-name" placeholder="Full name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Email address" required>
+            <textarea name="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500"
+              required></textarea>
+            <input type="submit" value="Get in touch">
+          </form>
+        </div>
         <div class="details">
           <div class="social">

```

Comment:
- [ ] You have done a great job on your contact section so far, but there one more thing we will need to fix here, see screenshots below
- Your contact section
![image](https://user-images.githubusercontent.com/55465133/129946051-0f893824-92d7-49c0-8d3e-0247316c28af.png)

- Figma contact section
![image](https://user-images.githubusercontent.com/55465133/129946139-697e1072-73a7-4630-958f-e95fe49cd77e.png)

We can see that you have some extra space from the left and right compared to my recent works section, they should actually have the same `margin-left` and `margin-right` to keep the design more consistent 😉